bindings/capacitorjs/ios/.gitignore

```diff
@@ -0,0 +1,3 @@
+libwallet.a
```

Comment:
Should this be `WalletFramework` instead? Since we're renaming `libwallet.a` to `WalletFramework` when we copy it over per the PR description

Reply:
you're right, thanks 👌